diff --git a/node_modules/@react-pdf/textkit/lib/textkit.js b/node_modules/@react-pdf/textkit/lib/textkit.js
index 7bfe0d9..268cae4 100644
--- a/node_modules/@react-pdf/textkit/lib/textkit.js
+++ b/node_modules/@react-pdf/textkit/lib/textkit.js
@@ -3063,8 +3063,8 @@ const scriptItemizer = () => {
                         end: index,
                         attributes: { script: lastScript },
                     });
+                    lastIndex = index;
                 }
-                lastIndex = index;
                 lastScript = script;
             }
             index += char.length;
